src/components/export/ExportTab/ExportTab.js

```diff
@@ -71,13 +78,47 @@ const ExportTab = ({
       workspaceSection
     );
   };
+
   const getInitSelection = (reportingPeriodObj) => {
     const { id, beginDate, endDate } = reportingPeriodObj;
     setReportingPeriod({ id, beginDate, endDate });
     if (exportState) {
-      setSelectedOptions({ beginDate, endDate });
+      setPreviewOptions({ beginDate, endDate });
     }
   };
+
+  const exportClickHandler = async () => {
+    let exportFileName
+    // export monitoring plan
+    if (dataTypes.find((e) => e.name === "monitoring-plan").checked) {
+      const mpRes = await getMonitoringPlanById(selectedConfig.id)
+      const facId = mpRes.data["facId"];
+      const mpName = mpRes.data["name"];
+      const date = new Date();
+      const year = date.getUTCFullYear();
+      const month = date.getUTCMonth() + 1;
+      const day = date.getUTCDate();
+      const fullDateString = `${month}-${day}-${year}`;
+      const facRes = await getFacilityById(facId)
+      const facName = facRes.data["facilityName"];
+      exportFileName = `MP Export - ${facName}, ${mpName} (${fullDateString}).json`;
+      download(JSON.stringify(mpRes.data, null, "\t"), exportFileName);
```

Comment:
You could have use `exportHandler` function from `src/components/HeaderInfo/HeaderInfo.js` instead for duplicating same code.

Reply:
moved export monitoring plan into exported function